index.html

```diff
@@ -13,53 +13,162 @@
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all3">
+        <img class="two" src="images/SnapshootPortfolio77.jpg" alt="">
+        <h2 class="multipost">Tonic
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words3" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="all4">
+        <img class="two" src="images/SnapshootPortfolio)1.jpg" alt="">
+        <h2 class="multipost"> Multi - Post
+            Stories
+        </h2>
+        <div class="list2">
+            <ul class="inline2">
+                <li><img class="words2" src="images/Frame2.jpg" alt=""></li>
+            </ul>
+        </div>
+        <h3 class="subtitle1">A daily selection is privaly 
+            personalized reads; 
+            no accounts or sign-ups required
+        </h3>
+        <div class="leng">
+            <ul>
+                <li><img class="lenguage" src="images/Tags.jpg" alt=""></li>
+                <li><a href="https://github.com/luccamicale"><img class="see" src="images/Enabled.jpg" alt=""></a></li>
+            </ul>
+        </div>
     </section>
-    <br>
-    <footer class="contac">
-        <h5>Lets Connect</h5>
 
-    </footer>
+    <section class="all5">
+        <h2 class="about"> About Myself </h2>
+        <img class="sub1" src="images/PrimaryText.jpg" alt="">
+        <h3 class="lets">LETS CONNECT</h3>
+        <div>
+            <ul>
+                <li> <a href="https://github.com/luccamicale"><img class="git1" src="images/Vectorgit.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letteri1" src="images/VectorI.png" alt=""></a></li>
+                <li> <a href="#"><img class="lettern1" src="images/Vectorn.jpg" alt=""></a></li>
+                <li> <a href="#"><img class="letterm1" src="images/Vectorm.jpg" alt=""></a></li>
+                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter1" src="images/Vectortw.jpg" alt=""></a></li>
+                <li> <a href=""><img class="hand1" src="images/Vectorhnad.jpg" alt=""></a></li>
+            </ul>
+        </div>
```

Comment:
Great job @luccamicale 🤝

- [ ] As specified in the project requirements, kindly align the contents of the about me section with `display: flex`.